### DISCUSSION TITLE
Backticks aren't handled the same way as in browser

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#110559] [Mobile] [answered]
GitHub Mobile beta for Android

- https://github.com/community/community/discussions/110517

![Screenshot_20240229-154127.png](https://github.com/community/community/assets/2119212/de5d9f2a-dcb3-4202-878d-b49ca1144abf)

![Screenshot_20240229-154237.png](https://github.com/community/community/assets/2119212/480fbeb6-77f7-406e-a2b5-b37f6d3f205d)

![Screenshot_20240229-154328.png](https://github.com/community/community/assets/2119212/5c1d0b73-f3b9-473d-b538-05b219287d5d)

Note: I personally don't want this case to be treated as `... `, but...

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref We have this in our backlog but haven't gotten to it yet. I marked your request. Thanks for writing in!

**Comment by buttatvkrabi-ui** [comment]
[README.md](https://github.com/user-attachments/files/24273891/README.md)
